packaging/nnstreamer.spec

```diff
@@ -251,14 +251,24 @@ NNStreamer filter subplugin for Verisicon Vivante.
 
 # for protobuf
 %if 0%{?protobuf_support}
-%package    nnstreamer-protobuf
+%package    -n nnstreamer-protobuf
```

Comment:
If you want the name `nnstreamer-protobuf`, what about simply removing `-n nnstreamer-`?

Reply:
Uh oh.. the most recommended one will be...
```
%package protobuf
```

Reply:
Right. I updated. Thank you!